src/Inviqa/Worldpay/Api/Request/PaymentService/Submit/Authorisation/AuthorisationOrder.php

```diff
@@ -39,7 +43,8 @@ public function xmlChildren()
             $this->description,
             $this->amount,
             $this->paymentDetails,
-            $this->shopper
+            $this->shopper,
+            $this->dynamic3DS
```

Comment:
`$this->dynamic3DS` could be omitted from the children list if it's not set.
This would avoid having to change the converted service to deal with nulls.